### PR TITLE
bugfix/307: Correct left spacing of hero on L2 pages

### LINKED ISSUE #307: Correct left spacing of hero on L2 pages
The left spacing of the hero on L2 pages (https://main--hlx-test--urfuwo.hlx.page/author/juliawhite) does not align with the elements below.

### PR DESCRIPTION
**Pull request by rjwtrmn**
Fix #307

Test URLs:

L2
- Before: https://main--hlx-test--urfuwo.hlx.page/draft/skhare/content-list-test
- After: https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.page/draft/skhare/content-list-test

L2, media blend
- Before: https://main--hlx-test--urfuwo.hlx.live/author/juliawhite
- After: https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.live/author/juliawhite

L1 (Unchanged)
- Before: https://main--hlx-test--urfuwo.hlx.live/draft/skhare/l1-sample-page
- After: https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.live/draft/skhare/l1-sample-page

L3 (Unchanged)
- Before: https://main--hlx-test--urfuwo.hlx.page/news/2024/01/accelerate-innovation-with-clean-core-strategy
- After: https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.page/news/2024/01/accelerate-innovation-with-clean-core-strategy

### CODE REVIEW

File: aemedge/styles/styles.css [hunk 1]
```diff
@@ -52,7 +52,8 @@
   --udexGridMargins: var(--udexGridXSMargins);
   --sapFontFamily: '72 Brand Variable', '72 Brand Variable Fallback', 'Arial',
     'Helvetica', 'sans-serif';
-  --hero-margin: var(--udexGridMargins);
+  --hero-margin--left: var(--udexGridMargins);
```
Comment (mhaack) [inline aemedge/styles/styles.css:55]: I know `--hero-margin` was there before, but setting extra variables based on predefined udex variables is maybe not a good choice. `--udexGridMargins` can be used in hero.css directly.

File: aemedge/styles/styles.css [hunk 2]
```diff
@@ -84,7 +85,8 @@
   :root {
     --udexGridGutters: var(--udexGridXLGutters);
     --udexGridMargins: var(--udexGridXLMargins);
-    --hero-margin: 246px;
+    --hero-margin--left: 246px;
```
Comment (mhaack) [inline aemedge/styles/styles.css:88]: We are already setting `--udexGridMargins` here which is used by the here, see above?

File: aemedge/styles/styles.css (hunk 1 above)
Comment (saurabh-khare) [reply aemedge/styles/styles.css:55]: @mhaack --hero-margin is set to multiple values not just --udexGridMargins.

File: aemedge/styles/styles.css (hunk 2 above)
Comment (saurabh-khare) [reply aemedge/styles/styles.css:88]: @mhaack We cannot set it to --udexGridMargins. We need 246px margins for XL while udexGridMargins = udexGridXLMargins for XL which is 84px

**Review by pahupe — CHANGES_REQUESTED** [review]
@rjwtrmn: In Chrome on Mac for viewport width 505px / XS (the narrowest my computer allows me to shrink to), on the following two pages the hero text block is wider (has more padding left/right) than the elements following the hero:
- https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.page/draft/skhare/content-list-test
- https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.live/author/juliawhite

All others look fine, also the L3 page with an in-page nav: https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.page/news/0000/the-article

File: aemedge/styles/styles.css (hunk 2 above)
Comment (mhaack) [reply aemedge/styles/styles.css:88]: Ok, got it.

**Comment by rjwtrmn** [comment]
> @rjwtrmn: In Chrome on Mac for viewport width 505px / XS (the narrowest my computer allows me to shrink to), on the following two pages the hero text block is wider (has more padding left/right) than the elements following the hero:
> 
> * https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.page/draft/skhare/content-list-test
> * https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.live/author/juliawhite
> 
> All others look fine, also the L3 page with an in-page nav: https://307-l2-page-hero-left-spacing--hlx-test--urfuwo.hlx.page/news/0000/the-article

Thank you, I have pushed a change to resolve the issue with the left padding. As regards the right side, I didn't make changes to the margin values there so it should match what is on main, but happy to change it here if main deviates from the designs (no Figma on my side as of yet though)

**Review by rootpea — APPROVED** [review]

**Review by mhaack — APPROVED** [review]

**Pull request merged**